### PR TITLE
[1876] fix(cookiecutter): Adds XML quote and backslashes escape support

### PR DESCRIPTION
**Pull request by Sharealikelicence**
Adds XML quote and backslashes escape support

[#1876](https://github.com/beeware/briefcase/issues/1876)

## PR Checklist:
- [x] All new features have been tested
- [x] All new features have been documented
- [x] I have read the **CONTRIBUTING.md** file
- [x] I will abide by the code of conduct

### CODE REVIEW

File: src/briefcase/integrations/cookiecutter.py [hunk 1]
```diff
@@ -139,6 +139,11 @@ def bool_attr(obj):
             """Render value in XML format appropriate for an attribute."""
             return "true" if obj else "false"
 
+        def escape_xml(obj):
+            """Escapes single quotes and backslashes."""
+            return obj.replace("\\", "\\\\").replace("'", "\\'")
```
Comment (kesara) [inline src/briefcase/integrations/cookiecutter.py:144]: May be best way to escape `'` and `"` is using `"'"` or `'"'`.
>To allow attribute values to contain both single and double quotes, the apostrophe or single-quote character (') may be represented as " &apos; ", and the double-quote character (") as " &quot; ".
See https://www.w3.org/TR/REC-xml/#syntax

Comment (kesara) [reply src/briefcase/integrations/cookiecutter.py:144]: So best thing to do here might be replacing `'` with `&apos;`.

Comment (kesara) [reply src/briefcase/integrations/cookiecutter.py:144]: Also https://docs.python.org/3/library/xml.sax.utils.html#xml.sax.saxutils.quoteattr might be of help.

**Comment by Sharealikelicence** [comment]
Unneeded change

**Pull request closed without merging**